frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -111,7 +117,7 @@ export default class DocumentListToolbar extends Component {
         <div class={styles['padded-group']}>
           <ButtonWithOptions
             accent="save"
-            disabled={disabled}
+            disabled={hasValidationErrors || disabled}
```

Comment:
That logic could be applied to `hasValidationErrors` too, as both simply disable the component. I think it makes sense to be clear on why the condition is true so would rather we keep both well labelled.